src/EditorFeatures/Test2/Rename/RenameEngineTests.CSharpConflicts.vb

```diff
@@ -11,7 +11,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
                 _outputHelper = outputHelper
             End Sub
 
-            <WpfFact(Skip:="799977")>
+            <WpfFact>
```

Comment:
You're talking about BreakingRenameWithRollBacksInsideLambdas_2? There's no "Foo" invocations in that test at all...